keadm/cmd/keadm/app/cmd/common/step_test.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2022 The KubeEdge Authors.
```

Comment:
since this is newly added, copyright year should be adjusted for current year.

```suggestion
Copyright 2025 The KubeEdge Authors.
```